### ISSUE TITLE
S3Sync Execution Fails

### ISSUE THREAD

**Issue by nypti-scott** [awslabs/aws-gov-cloud-import#16]
Trying to copy a bucket to GovCloud fails on the S3Sync task
It generates the following error in the EC2Worker logs: unable to sync: Error: ENOTCONN: socket is not connected, lstat '/home/ec2-user/s3fs/<commercial-bucket-name-removed>/'

The role permissions to access the S3 bucket match those recommended by the tool. 

I tested the mount script from a normal EC2 instance, and received a similar error. With S3FS debug level set to info, I received a number of the following messages:
s3fs[18127]: Pool empty: force to create new handler
s3fs[18127]: [IAM role=gov-cloud-import-ec2role-17FYW9CMQUPAL]

followed by a final: 
kernel: s3fs[18128]: segfault at 7f9f887a5f00 ip 00007f9f8d142737 sp 00007f9f887a5e20 error 6 in libc-2.26.so[7f9f8d0e7000+1ad000]

**Comment by awsjason** [comment]
Can you mount the S3 bucket via S3FS manually?  Then copy files manually to another local directory?

If you look at the mountBucket.sh, it will show you how its being mount so you can use the values.  

https://github.com/awslabs/aws-gov-cloud-import/blob/master/ec2/mountBucket.sh

**Comment by nypti-scott** [comment]
Manually mounting the S3 bucket with S3FS also fails. It seems to relate to using the IAM role for credentials. I was able to get it to mount when using a .passwd-s3fs file with credentials. 

I didn't find using the credentials file to be a workable solution for the import tool. However, I updated our deployed copy of gov-cloud-import-ec2prep.sh to use branch v1.84 of S3FS instead of the latest (v1.85). This let me complete the import without issue.
